src/components/common/DomListener.stories.mdx

```diff
@@ -35,6 +35,8 @@ When used as a root component, it will fire an action for any dom mutation and r
 </Preview>
 
 However be mindful that using it as a nested component will trigger DOM mutations only for children. In the following example clicking on buttons won't fire any event as the component is observing only the `+` button and no changes are made there. Resizing the window still works.
+^ This is a weird explanation. Were you trying to say that triggering an event without DOM changes will not fire `onUpdate`?
```

Comment:
not exactly, this component only listen for children modification and not for document modification:

```jsx
<SomeComponent />
<DomListener><AnotherComponent></DomListener>
```

any DOM changes happening in `SomeComponent` wont be caught by `DomListener`.

Reply:
That's intuitive right? Idk... 
Still this storybook component looks like a negative test to me. If it's a negative test I thought it should be in test and not showcased in storybook. Was very confused when I saw the component (esp when clicking wont work) until I read the code.

Reply:
as we discussed long time ago => will refactor by separating MutationObserver and window.resize listener